packages/wallets/src/components/WalletCard/WalletCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useWalletAccountsList } from '@deriv/api';
+import { WalletListCardBadge } from '../WalletListCardBadge';
+import './WalletCard.scss';
+
+type TProps = {
+    account: NonNullable<ReturnType<typeof useWalletAccountsList>['data']>[number];
+};
+
+const WalletCard: React.FC<TProps> = ({ account }) => {
+    return (
+        <div className='wallets-card' key={account.loginid}>
+            <div className='wallets-card__data'>
+                <div className='wallets-card__data__details'>
+                    <div className='wallets-card__data__details-icon'>
+                        <h1>{account.currency}</h1>
```

Comment:
Semantically, heading tags are for content. I dont think this h1 and the following h3 is suitable for this use case